linera-core/src/client.rs

```diff
@@ -2137,7 +2143,7 @@ impl ExecuteBlockOutcome {
     fn wait_for_timeout(info: &ChainInfo) -> Self {
         // TODO(#1424): Local timeout might not match validators' exactly.
         Self::WaitForTimeout(RoundTimeout {
-            timestamp: info.manager.round_timeout,
+            timestamp: info.manager.round_timeout.unwrap(),
```

Comment:
`.expect("some reason")` is preferred in production code

Reply:
Actually, this `unwrap()` is wrong!